apis/addons/v1alpha1/addons_types.go

```diff
@@ -284,6 +288,9 @@ type AddonStatus struct {
 	// Tracks last reported upgrade policy status.
 	// +optional
 	UpgradePolicy *AddonUpgradePolicyStatus `json:"upgradePolicy,omitempty"`
+	// Tracks the last reported addon status.
+	// +optional
+	ReportedStatus *OCMAddOnStatus `json:"reportedStatus,omitempty"`
```

Comment:
```suggestion
	OCMReportedStatus *OCMAddOnStatus `json:"reportedStatus,omitempty"`
```

The name `ReportedStatus` might not be meaningful enough for someone outside our team. Even now when I see it within the context of this PR, it took a second for the idea to sync in. 🧑‍💻 

 It would be great if we could indicate that this is the last status report to OCM. `OCMReportedStatus` is just my suggestion. I feel, at minimum we should somehow bring the word `OCM in this name. 🤔

Reply:
👍